package.json

```diff
@@ -151,5 +146,62 @@
     "webpack-assets-manifest": "3.1.1",
     "webpack-cli": "3.3.11",
     "webpack-merge": "4.2.2"
+  },
+  "engines": {
+    "node": "12"
+  },
+  "files": [],
+  "scripts": {
+    "build": "npm run clean && webpack --config webpack.config.js",
+    "circle:coverage": "nyc --reporter=lcov --reporter=html npm test -- --reporter mocha-junit-reporter --reporter-options mochaFile=reports/mocha/test-results.xml",
+    "circle:coveralls": "nyc report --reporter=text-lcov | coveralls",
+    "circle:lint": "npm run lint -- --format junit --output-file ./reports/eslint/eslint.xml",
+    "clean": "del .build/",
+    "contributors": "(git-authors-cli && finepack && git add package.json && git commit -m 'build: contributors' --no-verify) || true",
+    "coverage": "nyc --reporter=html --reporter=lcov npm test",
+    "coverage:open": "npm run coverage && open-cli ./coverage/index.html",
+    "develop": "run-p watch:js:*",
+    "lint": "eslint .",
+    "lint-staged": "lint-staged",
+    "postrelease": "npm run release:tags",
+    "prerelease": "npm run update:check && npm run contributors",
+    "release": "standard-version -a",
+    "release:github": "conventional-github-releaser -p angular",
+    "release:tags": "git push --follow-tags origin HEAD:master",
+    "start": "node start.js",
+    "test": "NODE_ENV=test LOG_LEVEL=silent mocha './app/**/*.test.js' './common/**/*.test.js' './config/**/*.test.js' --opts ./test/unit/mocha.opts",
+    "test-e2e:chrome": "testcafe chrome test/e2e/*.test.js",
+    "test-e2e:ci": "npm run test-e2e:chrome -- --color --reporter spec,xunit:reports/testcafe/results-chrome.xml",
+    "test-e2e:local": "E2E_BASE_URL='' npm run test-e2e:chrome -- --debug-on-fail",
+    "update": "ncu -u",
+    "update:check": "ncu -- --error-level 2",
+    "watch:js:client": "webpack --watch --progress",
+    "watch:js:server": "nodemon $NODE_DEBUG_OPTION --inspect",
+    "watch:test": "npm test -- --reporter=min --watch"
```

Comment:
We have lots of scripts. Maybe we should group them by theme, something like:
```
release: ...
release:pre: ...
```

Reply:
I think `prerelease` is actually a npm structure. When running `release` it will automatically run `prerelease` before the script starts and `postrelease` after it runs.